torch/distributed/distributed_c10d.py

```diff
@@ -989,6 +989,12 @@ def _is_barrier_after_init() -> int:
     return int(os.getenv("TORCH_DIST_INIT_BARRIER", "0"))
 
 
+def _abort_in_destroy_pg() -> bool:
+    # Environment variable to control whether to abort the communicators when users call destroy_process_group()
+    env = os.getenv("TORCH_NCCL_ABORT_IN_DESTROY_PG", "0")
+    return env == "1" or env.lower() == "true"
+
+
```

Comment:
Can this env ideally be avoided?
What are the pros and cons of defaulting to true?

Reply:
The env name may have a mismatch with the layer this operates, i.e. we are operating at python PG level here but the env name has NCCL in it.

Reply:
we need an env in order to have a way to do a smooth rollout

we can delete NCCL from the name but i think its better to keep it, since all of our PG envs are named `TORCH_NCCL`

Reply:
I agree this env is not ideal/needed in the long term.  this is the common tradeoff of long term maintenance burden (lot of legacy knobs) vs short term rollout safety. I think the compromise of both sides is to remove/cleanup those 'rollout' knobs as soon as the first few rollouts are safe and cleared